webapp-django/crashstats/crashstats/static/crashstats/css/base/tables.less

```diff
@@ -130,27 +17,126 @@ table {
             font-size: smaller;
         }
     }
-}
-tr {
-    z-index: 50;
-}
-table.tablesorter,
-table.zebra {
-    tbody {
-        tr:nth-child(odd) {
-            background-color: #edecec;
+
+    /* standard table styles for data tables */
+    &.data-table {
+        width: 100%;
+        th {
+            background-color: @dark-tan;
+        }
+        th,
+        td {
+            padding: .5em;
+        }
+        tbody {
+            background-color: @white;
+            tr:nth-child(odd) {
+                background-color: @tan;
+            }
+            tr:hover {
+                background-color: @light-grey;
+            }
+            tr.missingsymbols {
+                &,
+                &:hover,
+                & td {
+                    background-color: @red;
+                    color: @white;
+                    border-top: 2px solid @white;
+                }
+            }
         }
     }
-}
-
-/* truncated frames */
-table.data-table {
-    tbody {
-        tr.truncated-frame {
-            background-color: #FFC266;
+    &.captioned-data-table {
+        margin: 1rem;
+        width: 98%;
+        caption {
+            position: relative;
+            background-color: @tan;
+            padding: 1em 1rem .5rem;
+            .rounded-corners(6px, 6px);
+            h3 {
+                display: block;
+                margin: 0 0 .3em;
+                font-weight: bold;
+                font-style: normal;
+                font-size: 1.4rem;
+            }
             a {
-                color: #333;
+                &:link,
+                &:visited {
+                    color: @dark-grey;
+                }
+                &:focus {
+                    text-decoration: none;
+                }
+            }
+        }
+        th {
+            background-color: @grey;
+            color: @white;
+        }
+        th,
+        td {
+            padding: .5em;
+        }
+        tbody {
+            background-color: @white;
+            tr:nth-child(odd) {
+                background-color: @tan;
+            }
+            tr:hover {
+                background-color: @light-grey;
+            }
+        }
+
+        &.initially-hidden {
+            caption {
+                background-image: none;
+                .rounded-corners(6px);
+            }
+            thead {
+                .accessibly-hidden();
+            }
+            tbody {
+                .accessibly-hidden();
             }
         }
     }
-}
\ No newline at end of file
+
+    &.vertical {
+        th {
+            border: 1px solid @black;
+            font-weight: bold;
+        }
+    }
+
+
+    &.tablesorter,
+    &.zebra {
+        tbody {
+            tr:nth-child(odd) {
+                background-color: @tan;
+            }
+        }
+    }
+
+    /* truncated frames */
```

Comment:
Any reason not to nest this under the previous `&.data-table` selector?

Reply:
Nope, you're right. This has been moved up to the appropriate area.